src/Metadata/Property/Factory/AttributePropertyMetadataFactory.php

```diff
@@ -121,8 +121,23 @@ private function createMetadata(ApiProperty $attribute, ?ApiProperty $propertyMe
         }
 
         foreach (get_class_methods(ApiProperty::class) as $method) {
-            if (preg_match('/^(?:get|is)(.*)/', (string) $method, $matches) && null !== $val = $attribute->{$method}()) {
-                $propertyMetadata = $propertyMetadata->{"with{$matches[1]}"}($val);
+            if (preg_match('/^(?:get|is)(.*)/', (string) $method, $matches)) {
+                // BC layer, to remove in 5.0
+                if ('getBuiltinTypes' === $method) {
```

Comment:
why this? we can leave that be no?

Reply:
If removing that part, creating a `PropertyMetadata` from another will raise deprecations.